Darwin/Services/StitchRemoteMongoDBService/StitchSyncPerformanceTests/SyncR2LOnlyPerformanceTestDefinitions.swift

```diff
@@ -0,0 +1,213 @@
+import XCTest
+import Foundation
+import MongoSwift
+@testable import StitchRemoteMongoDBService
+@testable import StitchCoreRemoteMongoDBService
+//swiftlint:disable function_body_length
```

Comment:
[style nit] put this at the top of the file, and have a newline separate the imports and the class